src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const successHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
+  <div data-qa="notification" class="success">${message}</div>
+  `);
+};
+
+const errorHandler = (message) => {
+  body.insertAdjacentHTML('beforebegin', `
+  <div data-qa="notification" class="warning">${message}</div>
+  `);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  let clicked = false;
+
+  body.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      clicked = true;
+      resolve('First promise was resolved');
+    }
+  });
+
+  setTimeout(() => {
+    if (!clicked) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve) => {
+  document.body.addEventListener('click', (e) => {
+    resolve('Second promise was resolved');
+  });
```

Comment:
for second promise also should be listener for right button
![Screenshot from 2023-05-28 15-28-17](https://github.com/mate-academy/js_promises_practice_DOM/assets/33029864/b53801d7-e3e9-4ea4-925d-5937b9cffb10)